packages/admin/resources/lang/nl/resources/relation-managers/delete.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+return [
+
+    'action' => [
+
+        'label' => 'Verwijderen',
+
+        'modal' => [
+            'heading' => 'Verwijderen :label',
```

Comment:
I believe this should be `Verwijder :label`

Reply:
Good catch @bradyrenting! Thanks!